release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -407,16 +418,30 @@ In the following tables, features are marked with the following statuses:
 
 [id="ocp-4-16-removed-features"]
 === Removed features
-//.APIs removed from Kubernetes 1.27
-//[cols="2,2,2",options="header",]
-//|===
-//|Resource |Removed API |Migrate to
 
-//|`CSIStorageCapacity`
-//|`storage.k8s.io/v1beta1`
-//|`storage.k8s.io/v1`
+[id="ocp-4-16-removed-kube-1-29-apis"]
+==== Beta APIs removed from Kubernetes 1.29
+
+Kubernetes 1.29 removed the following deprecated API, so you must migrate manifests and API clients to use the appropriate API version. For more information about migrating removed APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-29[Kubernetes documentation].
+
+.APIs removed from Kubernetes 1.29
+[cols="2,2,2,1",options="header",]
+|===
+|Resource |Removed API |Migrate to |Notable changes
+
+|`FlowSchema`
+|`flowcontrol.apiserver.k8s.io/v1beta2`
+|`flowcontrol.apiserver.k8s.io/v1` or `flowcontrol.apiserver.k8s.io/v1beta3`
+|No
+
+|`PriorityLevelConfiguration`
+|`flowcontrol.apiserver.k8s.io/v1beta2`
+|`flowcontrol.apiserver.k8s.io/v1` or `flowcontrol.apiserver.k8s.io/v1beta3`
+|link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#flowcontrol-resources-v129[Yes]
```

Comment:
Could we name this link so that it aligns with the heading "notable changes"? If I see "notable changes," I expect more information than a simple "yes" or "no". Alternatively, the heading of the table could be something like "Contains notable changes".

Reply:
Good suggestions, but again I think I'll leave it be again for consistency's sake. (We also have KBase articles that use these same formats)